packages/x-charts/src/ChartsYAxis/ChartsYAxis.tsx

```diff
@@ -76,10 +72,37 @@ function ChartsYAxis(inProps: ChartsYAxisProps) {
     y: top + height / 2,
   };
 
-  const Line = slots?.axisLine ?? ChartsLine;
-  const Tick = slots?.axisTick ?? ChartsTick;
-  const TickLabel = slots?.axisTickLabel ?? ChartsTickLabel;
-  const Label = slots?.axisLabel ?? ChartsLabel;
+  const Line = slots?.axisLine ?? 'line';
+  const Tick = slots?.axisTick ?? 'line';
+  const TickLabel = slots?.axisTickLabel ?? Text;
+  const Label = slots?.axisLabel ?? Text;
+
+  const axisTickLabelProps = useSlotProps({
+    elementType: TickLabel,
+    externalSlotProps: slotProps?.axisTickLabel,
+    additionalProps: {
+      textAnchor: position === 'right' ? 'start' : 'end',
+      dominantBaseline: 'central',
+      style: { fontSize: tickFontSize },
+      className: classes.tickLabel,
+    } as const,
+    ownerState: {},
+  });
+
+  const axisLabelProps = useSlotProps({
+    elementType: Label,
+    externalSlotProps: slotProps?.axisTickLabel,
+    additionalProps: {
+      textAnchor: 'middle',
+      dominantBaseline: 'central',
+      style: {
+        fontSize: labelFontSize,
+        transform: `rotate(${positionSigne * 90}deg)`,
+        transformOrigin: `${labelRefPoint.x}px ${labelRefPoint.y}px`,
+      },
```

Comment:
Yes, I was in the middle of a thinking about
- drop `className` since the style should be passe to the `<ChartsText/>` props
- keep it for colors, and other stuff that do not impact size